diff --git a/src/verso-literate-html/LiterateHtmlMain.lean b/src/verso-literate-html/LiterateHtmlMain.lean
--- a/src/verso-literate-html/LiterateHtmlMain.lean
+++ b/src/verso-literate-html/LiterateHtmlMain.lean
@@ -2,6 +2,6 @@ private def renderModBody (mod : LitMod) (resolved : ResolvedConfig)
   let emitCtx := { ctx with
     options := {}
     traverseContext := { currentModule := mod.name }
-    codeOptions := {}
+    codeOptions := { inlineProofStates := false }
   }
   let hlCtx : HighlightHtmlM.Context Literate := { emitCtx with options := emitCtx.codeOptions }
